pom.xml

```diff
@@ -1890,17 +1890,6 @@
 
   <profiles>
 
-    <!-- Till https://github.com/powermock/powermock/issues/1094 -->
```

Comment:
It looks like new issue was not necessary.

Reply:
Please confirm if this issue can be closed when this is merged.

Reply:
it is issue not in our repo.
I think it is ok to keep #11067 .

Reply:
My bad. I saw issue and did not look at it closer. Resolving chain.